src/Components/CurrentCommand/CurrentCommand.js

```diff
@@ -180,6 +179,7 @@ function Footer({ config, orders, setOrders, setConfig, price, priceLess, payLis
             .catch(error => {
                 console.log(error);
             });
+            data.id = data.sequence_value;
```

Comment:
Missing a small modification from the last changes requested.
```suggestion
            data.id = data.orderId;
```